### PR TITLE
fix(submit): incorrect file arguments processing 

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: tests/test_regression.py [hunk 1]
```diff
@@ -157,3 +157,32 @@ def test_which_universum_is_tested(docker_main, pytestconfig):
...
+                                       "-cm", "This is change", "-vt", "p4"],
+                              environment = {{"P4CLIENT": "{perforce_environment.client_name}",
+                                              "P4PORT": "{perforce_environment.p4.port}",
+                                              "P4USER": "{perforce_environment.p4.user}",
+                                              "P4PASSWD": "{perforce_environment.p4.password}"}})])
+"""
+    settings = shelve_config(config, perforce_environment)
+
+    assert not __main__.run(settings)
+    stdout_checker.assert_absent_calls_with_param("Submit - \x1b[1;31mFailed\x1b[00m")  # terminal coloring included
```
Comment (i-keliukh) [inline tests/test_regression.py:186]: assert_absent_calls_with_param support regular expressions. You don't need to embed escape sequences.

File: universum/submit.py [hunk 2]
```diff
@@ -34,17 +34,16 @@ def define_arguments(parser):
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
-        self.read_and_check_multiline_option("commit_message", """
+        self.commit_message = self.read_and_check_multiline_option("commit_message", """
```
Comment (i-keliukh) [inline universum/submit.py:37]: Is there test for commit message?

File: tests/test_submit.py [hunk 3]
```diff
@@ -360,3 +360,31 @@ def test_success_reconcile_wildcard(submit_parameters, submit_environment):
...
+
+
+def test_success_reconcile_partial(submit_parameters, submit_environment):
+    parameters = submit_parameters(submit_environment)
+
+    dir_name = utils.randomize_name("new_directory")
+
+    tmp_dir = parameters.environment.vcs_cooking_dir.mkdir(dir_name)
+    for i in range(0, 9):
+        tmp_file = tmp_dir.join("new_file{}.txt".format(i))
```
Comment (i-keliukh) [inline tests/test_submit.py:372]: In my opinoin, f-string would look better here

File: tests/test_submit.py [hunk 4]
```diff
@@ -360,3 +360,31 @@ def test_success_reconcile_wildcard(submit_parameters, submit_environment):
     parameters.assert_submit_success([str(parameters.environment.vcs_cooking_dir) + "/*"])
     assert not parameters.file_present(str(tmp_dir))
     assert not parameters.file_present(str(other_tmp_dir))
+
+
+def test_success_reconcile_partial(submit_parameters, submit_environment):
```
Comment (i-keliukh) [inline tests/test_submit.py:365]: Please also document what issue is verified by this test.

**Review by axxie — APPROVED** [review]

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**